### PR TITLE
feat: redesign error 404 page

### PR DESCRIPTION
**Pull request by MoazMirza-13**
[Page Not Found _ Electron.webm](https://github.com/electron/website/assets/134006296/c36a22ae-0d9e-40df-8f3d-7e932149f070)

Redesigned the 404 error page.
fixed the https://github.com/electron/website/issues/194
You can see in the video.

close: #194

### CODE REVIEW

**Review by BlackHole1 — APPROVED** [review]
LGTM. PTAL @erickzhao

File: src/theme/notFound.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```
Comment (erickzhao) [inline src/theme/notFound.scss:1]: I don't think we should be adding a new font just for this page. Let's just use the existing styles.

**Review by erickzhao — CHANGES_REQUESTED** [review]

File: src/theme/notFound.scss (hunk 1 above)
Comment (MoazMirza-13) [reply src/theme/notFound.scss]: as requested I removed the fonts.

**Comment by dsanders11** [comment]
What is the source of the image added in this PR? I want to make sure we're allowed to use it.

**Comment by MoazMirza-13** [comment]
> What is the source of the image added in this PR? I want to make sure we're allowed to use it.

I'm sorry I didn't get what you are trying to ask. Are you asking about the video that I attached with this pr or are you asking about dead-computer-icon used in this pr ?

**Comment by dsanders11** [comment]
> are you asking about dead-computer-icon used in this pr ?

Yes

**Comment by MoazMirza-13** [comment]
> > are you asking about dead-computer-icon used in this pr ?
> 
> Yes

Well, When i started to work on this issue, I saw that someone else tried to do it too and he raised his pr year ago. (link below);
https://github.com/electron/website/pull/257
but he had issues with his pr and now it's been more than a year, so i almost used his code to redesign it and got this image from his pr too. (link below)
https://raw.githubusercontent.com/electron/website/64adcedb673f37db9acbf87266ca296b28fa9bcc/static/assets/img/dead-computer-icon.webp

**Comment by Kilian** [comment]
This image can **not** be used without a license and needs to be bought.

https://www.iconfinder.com/icons/2202521/computer_dead_desktop_emoji_monitor_screen_troubleshoot_icon

**Review by Kilian — CHANGES_REQUESTED** [review]
We do not have the rights to use the provided image.

**Comment by MoazMirza-13** [comment]
> We do not have the rights to use the provided image.

@Kilian can you tell me which image should I use ?

**Comment by MoazMirza-13** [comment]
> We do not have the rights to use the provided image.

can we use this (it has a free license);

https://www.freepik.com/free-vector/page-found-concept-illustration_7887410.htm#query=404%20page%20found&position=4&from_view=keyword&track=ais

**Comment by BlackHole1** [comment]
> > We do not have the rights to use the provided image.
> 
> can we use this (it has a free license);
> 
> https://www.freepik.com/free-vector/page-found-concept-illustration_7887410.htm#query=404%20page%20found&position=4&from_view=keyword&track=ais

Yes. We can use it. 👍

**Comment by Kilian** [comment]
> > > We do not have the rights to use the provided image.
> > 
> > 
> > can we use this (it has a free license);
> > https://www.freepik.com/free-vector/page-found-concept-illustration_7887410.htm#query=404%20page%20found&position=4&from_view=keyword&track=ais
> 
> Yes. We can use it. 👍

...provided we add attribution. I'm not sure that's something we want to do on the website.

**Comment by BlackHole1** [comment]
@Kilian Maybe I missed something, but I didn’t see anything above stating that attribution is required for using this image. 🤔

14-25 16 14@2x

**Comment by BlackHole1** [comment]
I asked a friend who is a designer to help me design a 404 image, with no restrictions. We can use it directly. 😃

Download: [404.zip](https://github.com/electron/website/files/13163577/404.zip)

Preview:
57-25 16 57@2x

**Comment by MoazMirza-13** [comment]
is it good now ?

[Page Not Found _ Electron.webm](https://github.com/electron/website/assets/134006296/ce8ab8cd-f215-41a2-a146-4031deab2ddb)

**Comment by BlackHole1** [comment]
LGTM. PTAL @dsanders11 @Kilian

**Comment by BlackHole1** [comment]
Ping @dsanders11 @Kilian

**Review by erickzhao — COMMENTED** [review]
Hey @MoazMirza-13, thanks for the contribution. Going to fix some of these issues up myself and land this PR today.

**Review by erickzhao — APPROVED** [review]

**Pull request merged**